Documentation/ApiOverview/Hooks/EventDispatcher/Index.rst

```diff
@@ -26,6 +26,71 @@ Benni Mack: "Don't get hooked, listen to events! PSR-14 within TYPO3 v10" @ TYPO
   Additional background on the implementation can be found at https://usetypo3.com/psr-14-events.html
 
 
+.. _EventDispatcherQuickStart:
+
+Quick start
+===========
+
+.. _EventDispatcherQuickStartDispatching:
+
+Dispatching an event
+--------------------
+
+#. Create an event class. 
+
+   An event class is basically a plain PHP object with getters for immutable
+   properties and setters for mutable properties. It contains a constructor for all properties::
+   
+      class DoingThisAndThatEvent {
+         private string mutableProperty;
+         private int inmutableProperty;
+         
+         public function __construct(string mutableProperty, int inmutableProperty) {
+            // ...
+         }
+         
+         // Getter for both properties, setters only for $mutableProperty;
+      }
+      
+   Read more about :ref:`implementing event classes <EventDispatcherEvents>`.
+      
+#. Inject the EventDispatcher
+
+   If you are in a Controller the EventDispatcher already got injected in the ActionController
+   in this case you can ommit this step.
+   
+   If the EventDispatcher is not yet available, you need inject it::
+   
+      class MyClass {
+         
+          use Psr\EventDispatcher\EventDispatcherInterface;
+         
+          /**
+           * @var EventDispatcherInterface
+           */
+          private $eventDispatcher;
+          
+          public function injectEventDispatcher(EventDispatcherInterface $eventDispatcher): void
+          {
+              $this->eventDispatcher = $eventDispatcher;
+          }
+      }
+      
+#. Dispatch the event
+
+   Create an event object with the data that should be passed to the listeners. Use the data 
+   of mutable properties however it suits your business logic::
+   
+   public doSomething() {
+      // ..
+      /** @var DoingThisAndThatEvent $event */
+      $event = $this->eventDispatcher->dispatch(
+         new DoingThisAndThatEvent("foo", "bar")
+      );
+      $someChangedValue = $event->getMutableProperty();
```

Comment:
We should explicitly add this getter (and a setter) to the event above.

Reply:
I think every developer should know what a getter and setter look like. Most ides generate them automatically. And it would only blow up the quick start example

Reply:
Indeed. I usually just like to have everything explicit so that even the copypasta people have some success. ;-)